src/store/auth/reducer.js

```diff
@@ -90,6 +90,7 @@ export const authReducer = (state = INITIAL_STATE, action) => {
         loginError: null,
         signingIn: false,
         fbError: '',
+        isRefreshingTokens: false,
```

Comment:
(is this used?)

Reply:
This makes it so that variable doesn't persist on app launches.
It's not used but in the future it may be used so might as well keep it there.